lib/fastlane/plugin/wpmreleasetoolkit/actions/common/setfrozentag_action.rb

```diff
@@ -71,7 +71,7 @@ def self.available_options
                                        description: 'The GitHub milestone',
```

Comment:
Could be worth fixing this `ConfigItem`'s `description` while at it (seems like this was an accidental copy/paste from the previous one)

Reply:
Fixed here: https://github.com/wordpress-mobile/release-toolkit/pull/469/commits/cb40b83a2c59034795d754980ab00bfe3949644d